.github/workflows/release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           while curl -sf -o /dev/null "https://api.github.com/repos/$GITHUB_REPOSITORY/git/ref/tags/$DATE_PART.$PATCH_VERSION"; do
             (( PATCH_VERSION++ ))
```

Comment:
@mattrobenolt for some reason this script still gives an error when run on GHA but not locally. Any ideas?

Reply:
No, but there’s a dozen ways to increment a variable: https://askubuntu.com/questions/385528/how-to-increment-a-variable-in-bash#385532

I’d probably just refactor this into a normal for..loop instead with a max patch version. 

Like

```bash
for patch in {0..20}; do
  ...
```

Reply:
Do you think it is worth trying to cover this edge case? What would be the max patch number anyway, 100? 500?